include/zephyr/net/net_pkt.h

```diff
@@ -483,25 +492,6 @@ static inline void net_pkt_set_ip_ecn(struct net_pkt *pkt, uint8_t ecn)
 #endif
 }
 
-static inline uint8_t net_pkt_tcp_1st_msg(struct net_pkt *pkt)
```

Comment:
This is a public API and should probably follow deprecation guidelines.

Reply:
There are no one in the stack that calls it. Originally it was used by the TCP for its internal housekeeping but for some reason it was not removed when the calls in tcp.c were removed. I think we can remove it safely at this point.

Reply:
This should anyway go into another PR, it's unrelated to ethernet afaik.

Reply:
The TCP obsolete bit removal commit is now in #71432